src/containers/UnpublishedResourcePage/UnpublishedResourcePage.tsx

```diff
@@ -7,32 +7,42 @@
  */
 
 import { useTranslation } from "react-i18next";
+import {
+  ErrorMessageDescription,
+  ErrorMessageRoot,
+  ErrorMessageTitle,
+  ErrorMessageContent,
+  ErrorMessageActions,
+} from "@ndla/primitives";
+import { SafeLink } from "@ndla/safelink";
+import { styled } from "@ndla/styled-system/jsx";
 import { HelmetWithTracker } from "@ndla/tracker";
-import { OneColumn, ErrorMessage } from "@ndla/ui";
 import { Status } from "../../components";
 
+const StyledErrorMessageRoot = styled(ErrorMessageRoot, {
```

Comment:
Går det an å bruke `PageContainer` fremfor denne?

Reply:
Altså istedenfor `ErrorMessageRoot`?

Reply:
Nei, istedenfor margin!

Reply:
Da får den ikke marginen på toppen sånn som designet har da?
margin:
![image](https://github.com/user-attachments/assets/45ca737b-1306-4ebe-a013-da81461603a5)


vs PageContainer
![image](https://github.com/user-attachments/assets/fb466d73-1f6a-40f2-9265-78ead7a1eeb3)


(for meg så er det ikke så farlig)

Reply:
Aha det finnes to `PageContainer`.
Det funker!